adrs/0000-template.md

```diff
@@ -0,0 +1,16 @@
+# [ADR Title]
+
+**Date:** [YYYY-MM-DD]
+
+**Status:** [ 🚧 Proposed | ✅ Accepted | 🚫 Rejected ]
+
+**Related PR:** [#xxxx (when applicable)]
```

Comment:
I don't mind having this here in general, but we should also ensure that the PR number is in the commit message (following the standard already laid out in Stacks-Editor w/ conventional commits)

Reply:
I think it's important to have this here for people browsing the markdown directly. It might not be immediate for people that additional information is available in the PR linked in the commit message.

Reply:
Sounds good to me! Agreed that having it here is not terribly redundant and greatly improves visibility. I think having it in both places is totally fine.